exercises/grep/grep_test.exs

```diff
@@ -0,0 +1,259 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("bob.exs", __DIR__)
```

Comment:
hahah, obviously, "I'm gonna fix that after I've written the tests", duh... sorry :(